include/assimp/MemoryIOWrapper.h

```diff
@@ -167,6 +162,7 @@ class MemoryIOSystem : public IOSystem {
     // -------------------------------------------------------------------
     /** Tests for the existence of a file at the given path. */
     bool Exists(const char* pFile) const override {
+        printf("Exists\n");
```

Comment:
You probably didn't mean to commit this.

Reply:
Right :-).